### ISSUE TITLE
Added support for TCP_QUICKACK

### ISSUE THREAD

**Issue by JCash** [defold/extension-websocket#3]
There's now a function in the dmSDK ([dmSocket::SetQuickAck()](https://defold.com/ref/beta/dmSocket/#dmSocket::SetQuickAck:socket-use_quick_ack)) that sets this flag on the supported platforms.

The question is where and when to call this function, since the flag seems to be reset every now and then.
E.g. according to this [reference](https://github.com/urllib3/urllib3/issues/746#issuecomment-293965128)

**Comment by britzl** [comment]
Yes, I saw the same on SO: https://stackoverflow.com/a/1615591/1266551

"It appears that TCP_QUICKACK is only reset when there is data being sent or received; if you're not sending any data, then it will only get reset when you receive data, in which case you can simply set it again."
